src/sage/combinat/combination.py

```diff
@@ -47,6 +47,17 @@ class of combinations of ``mset`` of size ``k``.
     The combinatorial classes correctly handle the cases where ``mset`` has
     duplicate elements.
 
+    INPUT:
+
+    - ``mset`` -- a list, tuple, set, or integer `n` (interpreted as ``range(n)``)
+    - ``k`` -- (optional) an integer, the size of the combinations
+    - ``allow_negative`` -- (default: ``None``) boolean or ``None``. Controls
+      behavior when ``k`` is negative.
+      - ``None``: Return an empty list and issue a deprecation warning (:issue:`39411`).
+        This will change in the future to raise a ``ValueError``.
+      - ``True``: Return an empty list without a warning.
+      - ``False``: Raise a ``ValueError``.
```

Comment:
```suggestion

      * ``None``: Return an empty list and issue a deprecation warning (:issue:`39411`).
        This will change in the future to raise a ``ValueError``.
      * ``True``: Return an empty list without a warning.
      * ``False``: Raise a ``ValueError``.
```
The documentation fails to build, but I think this will fix the error.